mumbleBot.py

```diff
@@ -776,6 +784,8 @@ def start_web_interface(addr, port):
                         type=str, default=None, help="Certificate file")
     parser.add_argument("-b", "--bandwidth", dest="bandwidth",
                         type=int, help="Bandwidth used by the bot")
+    parser.add_argument("-B", "--bots", dest="bots",
```

Comment:
I would not create an individual command-line option just for this function. After some point, we stop adding new features into a command-line option because there are too many of them and it would be better just to put them into a config file.